File: Herança - Subclasses de Personagem/Item.py
class Item:
    def __init__(self, n="Sem nome", b=0):
        self.nome = n
        self.setBonus(b)

    def getNome(self):
        return self.nome

    def getBonus(self):
        return self.bonus

    def setNome(self, n):
        self.nome = n

    def setBonus(self, b):
        if b >= 0:
            self.bonus = b

    def descricao(self):
        return self.getNome() + " (+" + str(self.getBonus()) + ")"

File: Polimorfismo - Batalha Final/Item.java
public class Item {
    private String nome="Sem nome";
    private int bonus;

    public Item(){

    }
    public Item(String n, int b){
        setNome(n);
        setBonus(b);
    }

    public String getNome(){
        return this.nome;
    }
    public int getBonus(){
        return this.bonus;
    }
    public void setNome(String n){
        this.nome = n;
    }
    public void setBonus(int b){
        if (b>=0){
            this.bonus = b;

        }
    }

    public String descricao(){
        return getNome() + " (+" + getBonus() + ")";
    }
}